packages/react-native-mmkv/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@dev-plugins/react-native-mmkv",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

```suggestion
  "version": "0.4.0",
```

kindly keep it as it was and we'll see how to bump it